core/services/ocr2/database.go

```diff
@@ -25,28 +27,38 @@ var (
 )
 
 // NewDB returns a new DB scoped to this oracleSpecID
-func NewDB(sqldb *sql.DB, oracleSpecID int32, lggr logger.Logger) *db {
-	return &db{sqldb, oracleSpecID, lggr}
+func NewDB(sqlxDB *sqlx.DB, oracleSpecID int32, lggr logger.Logger, cfg pg.LogConfig) *db {
+	namedLogger := lggr.Named("OCR2 DB")
```

Comment:
I think we want to avoid spaces here. Maybe `OCR2.DB`? (Or set `OCR2` up the stack, and just `DB` here?)

Reply:
yeah, makes sense